### PR TITLE
Deprecating Attach Panel APIs

### PR DESCRIPTION
**Pull request by tarunkumarmahay1999**
Signed-off-by: tarun.mahay <tarun.mahay@samsung.com>

### Description of Change ###
This patch deprecates Tizen.Applications.AttachPanel due to ElmSharp deprecation.

### API Changes ###

 - ACR: TCSACR-519

### CODE REVIEW

File: src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public partial class AttachPanel
         /// <exception cref="OutOfMemoryException">Thrown when an attempt to allocate the memory fails.</exception>
```
Comment (wiertel) [inline src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs:18]: please add the Obsolete tag for AttachPanel (line 9) too

File: src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/StateEventArgs.cs [hunk 2]
```diff
@@ -19,6 +19,7 @@ internal StateEventArgs(EventType eventType)
         /// Property for the event type.
```
Comment (wiertel) [inline src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/StateEventArgs.cs:19]: please add Obsolete attribute for the StateEventArgs class too (line 9)

File: src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs (hunk 1 above)
Comment (tarunkumarmahay1999) [reply src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanel.cs:19]: Ack

**Comment by tarunkumarmahay1999** [comment]
Modified the required changes

**Review by tarunkumarmahay1999 — COMMENTED** [review]
Updated the required changes

**Review by WonyoungChoi — CHANGES_REQUESTED** [review]
Missing `Obsolete` attribute for `public class ResultEventArgs`.

**Review by WonyoungChoi — APPROVED** [review]

**Pull request merged**